src/portable/raspberrypi/rp2040/hcd_rp2040.c

```diff
@@ -537,7 +537,11 @@ bool hcd_setup_send(uint8_t rhport, uint8_t dev_addr, uint8_t const setup_packet
     (void) rhport;
 
     // Copy data into setup packet buffer
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds"
+#pragma GCC diagnostic ignored "-Wstringop-overflow"
```

Comment:
this warnings from gcc11 is bit annoying, hopefully we could find an better way to go around this.